repositories/src/test/scala/cromwell/pipeline/datastorage/dao/repository/utils/TestProjectUtils.scala

```diff
@@ -1,19 +1,18 @@
 package cromwell.pipeline.datastorage.dao.repository.utils
 
-import java.util.UUID
-
-import cromwell.pipeline.datastorage.dto._
+import cromwell.pipeline.datastorage.dto.{ Project, ProjectId }
+import cromwell.pipeline.model.wrapper.UserId
 
 object TestProjectUtils {
+  val userUuid = UserId.random
+  val projectUuid = java.util.UUID.randomUUID().toString
```

Comment:
Why not keep them `def` instead of `val`? You won't have to provide custom `ProjectId` everywhere in tests.

Reply:
Done